pkg/util/util.go

```diff
@@ -140,6 +143,20 @@ func ConvertToSingleDocs(doc []byte) [][]byte {
 	return singleDocs
 }
 
+func YAMLStringToSingleDocs(doc string) []string {
```

Comment:
can't we just convert the input to bytes and use the above function instead of having to duplicate this entire function?

Reply:
made some minor changes to calling functions to make this happen in https://github.com/replicatedhq/kots/pull/4006/commits/054efb746ec00582dbc882c836dbd0bde8905bee